pkg/cmd/roachprod/main.go

```diff
@@ -2109,3 +2110,19 @@ The logs will be placed in the directory if specified or in the directory named
 			fetchLogsTimeout)
 	}),
 }
+
+var getLatestPprof = &cobra.Command{
```

Comment:
Nit: For capitalisation consistency when `pprof` >= 2nd word, I noticed `workloadPProfStartPort`. Just so we don't have a mix of `Pprof` and `PProf` in the code.